public/js/components/Editor.js

```diff
@@ -26,6 +26,41 @@ function isTextForSource(sourceText) {
   return !sourceText.get("loading") && !sourceText.get("error");
 }
 
+function breakpointAtLine(breakpoints, line) {
+  return breakpoints.find(b => {
+    return b.location.line === line + 1;
+  });
+}
+
+function renderConditionalBreakpointPanel(
+  { location, setBreakpointCondition, condition, closePanel }) {
+  function onKey(e) {
+    if (e.key != "Enter") {
+      return;
+    }
+
+    setBreakpointCondition(location, e.target.value);
+    closePanel();
+  }
+
+  let panel = document.createElement("div");
+  ReactDOM.render(
```

Comment:
we likely can replace the react rendering with a simple vanilla approach, but on the other hand this *does* work.